### PR TITLE
ES2015 and promisify

### PR DESCRIPTION
**Pull request by kevva**
Also moves `osx-vol` and `linux-vol` into the module. Removed Travis too since it didn't work anyway. I would extract the CLI to a separate module before bumping a new version though.

### CODE REVIEW

**Review by sindresorhus — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +0,0 @@
-sudo: false
-language: node_js
-node_js:
-  - 'iojs'
-  - '0.12'
-  - '0.10'
```
Comment (sindresorhus) [inline .travis.yml:6]: Intentional?

Comment (gillstrom) [reply .travis.yml:6]: I  think so. The test included a Volkswagen line: `if (!process.env.CI) {` :(

Comment (sindresorhus) [reply .travis.yml:6]: Ok, then this repo should be disabled in the Travis settings.

Comment (gillstrom) [reply .travis.yml:6]: Yepp, done.

**Review by gillstrom — APPROVED** [review]

**Comment by gillstrom** [comment]
Thanks! I will extract the CLI later today.

File: .travis.yml (hunk 1 above)
Comment (kevva) [reply .travis.yml:6]: Intentional, yes, since Travis doesn't have sound.

**Pull request merged**